### ISSUE TITLE
Logs Explorer icon in Activity bar should have bolder borders

### ISSUE THREAD

**Issue by VKondakoff** [axilleasiv/vscode-javascript-repl-docs#8]
I think Logs Explorer icon in Activity bar should have bolder borders. They are rather thin now (compared with other icons):

<img width="46" alt="Image on 2019-11-14 04 23 20 PM" src="https://user-images.githubusercontent.com/230824/68860938-8a980d00-06fb-11ea-843e-c2d638f801b1.png"> <img width="46" alt="Image on 2019-11-14 04 23 09 PM" src="https://user-images.githubusercontent.com/230824/68860941-8d92fd80-06fb-11ea-9ed3-b0d74caa1ccd.png">

**Comment by axilleasiv** [comment]
Good catch! I really liked this SVG icon but is the truth that it is thinner.
